articles/aks/rdp.md

```diff
@@ -23,128 +23,63 @@ This article assumes that you have an existing AKS cluster with a Windows Server
-
-AKS node pool subnets are protected with NSGs (Network Security Groups) by default. To get access to the virtual machine, you'll have to enabled access in the NSG.
-
-> [!NOTE]
-> The NSGs are controlled by the AKS service. Any change you make to the NSG will be overwritten at any time by the control plane.
->
-
-First, get the resource group and nsg name of the nsg to add the rule to:
-
-```azurecli-interactive
-CLUSTER_RG=$(az aks show -g myResourceGroup -n myAKSCluster --query nodeResourceGroup -o tsv)
-NSG_NAME=$(az network nsg list -g $CLUSTER_RG --query [].name -o tsv)
+kubectl port-forward node-debugger-aks-nodepool1-12345678-vmss000000-bkmmx 2022:22
+Forwarding from 127.0.0.1:2022 -> 22
+Forwarding from [::1]:2022 -> 22
 ```
 
-Then, create the NSG rule:
+Open a third terminal window and use `kubectl get nodes` to show the internal IP address of the Windows Server node:
 
-```azurecli-interactive
-az network nsg rule create --name tempRDPAccess --resource-group $CLUSTER_RG --nsg-name $NSG_NAME --priority 100 --destination-port-range 3389 --protocol Tcp --description "Temporary RDP access to Windows nodes"
+```output
+$ kubectl get nodes -o wide
+NAME                                STATUS   ROLES   AGE     VERSION   INTERNAL-IP   EXTERNAL-IP   OS-IMAGE                         KERNEL-VERSION     CONTAINER-RUNTIME
+aks-nodepool1-12345678-vmss000000   Ready    agent   13m     v1.19.9   10.240.0.4    <none>        Ubuntu 18.04.5 LTS               5.4.0-1046-azure   containerd://1.4.4+azure
+aks-nodepool1-12345678-vmss000001   Ready    agent   13m     v1.19.9   10.240.0.35   <none>        Ubuntu 18.04.5 LTS               5.4.0-1046-azure   containerd://1.4.4+azure
+aksnpwin000000                      Ready    agent   87s     v1.19.9   10.240.0.67   <none>        Windows Server 2019 Datacenter   10.0.17763.1935    docker://19.3.1
 ```
 
-## Get the node address
-
-To manage a Kubernetes cluster, you use [kubectl][kubectl], the Kubernetes command-line client. If you use Azure Cloud Shell, `kubectl` is already installed. To install `kubectl` locally, use the [az aks install-cli][az-aks-install-cli] command:
-    
-```azurecli-interactive
-az aks install-cli
-```
+In the above example, *10.240.0.67* is the internal IP address of the Windows Server node.
 
-To configure `kubectl` to connect to your Kubernetes cluster, use the [az aks get-credentials][az-aks-get-credentials] command. This command downloads credentials and configures the Kubernetes CLI to use them.
+Use ssh to forward the RDP port of the Windows machine to your local network (you may need to change the first local port 3389 if it is being used on your computer):
 
-```azurecli-interactive
-az aks get-credentials --resource-group myResourceGroup --name myAKSCluster
 ```
-
-List the internal IP address of the Windows Server nodes using the [kubectl get][kubectl-get] command:
-
-```console
-kubectl get nodes -o wide
+ssh -p 2022 -L 3389:10.240.0.67:3389 -A azureuser@127.0.0.1
```

Comment:
If private key is not set in ~/.ssh/.ssh, below error will be thrown:
```
Authorized uses only. All activity may be monitored and reported.
azureuser@127.0.0.1: Permission denied (publickey).
```

Reply:
yes, You have to have the correct ssh key configured in you aks cluster. You can try resetting it via portal if you used a generated key.